### PR TITLE
Updated molecular test yaml

### PR DESCRIPTION
**Pull request by wwysoc2**
https://jira.opensciencedatacloud.org/browse/DICT-233

### CODE REVIEW

File: gdcdictionary/schemas/molecular_test.yaml [hunk 1]
```diff
@@ -6980,6 +7015,9 @@ properties:
       - Test Value Reported
       - Unknown
       - Not Reported
+      - Elevated
+      - Amplified
+      - Not Amplified
```
Comment (bullenc) [inline gdcdictionary/schemas/molecular_test.yaml:7020]: I think these need to be alphabetized?

**Review by bullenc — CHANGES_REQUESTED** [review]

File: gdcdictionary/schemas/molecular_test.yaml (hunk 1 above)
Comment (bullenc) [reply gdcdictionary/schemas/molecular_test.yaml]: 🙌

**Review by sadhukd — APPROVED** [review]
Looks good.

**Review by wteouchicago — APPROVED** [review]
looks good

**Review by bullenc — COMMENTED** [review]
Looks great 👍

**Review by mpsolano — APPROVED** [review]

**Review by tlicht3 — APPROVED** [review]

**Review by bullenc — APPROVED** [review]

**Pull request merged**